### DISCUSSION TITLE
Expand Document Section

### DISCUSSION THREAD

**Discussion by nicolinux** [outline/outline#6551] [Ideas]
Please add the feature to expand a text section. This allows for a tidier (and easiert to read) flow of text. Doesn't need to be fancy, something like the `/expand` macro in Confluence would be fine.
![expand](https://github.com/outline/outline/assets/1069286/a097aa58-fd2a-441d-ae20-f8234b188eba)

**Comment by notfounnd** [comment]
Upvoting here.

**Comment by atif1001** [comment]
Yes, it will be really nice to have this feature.
